### DISCUSSION TITLE
Is this possible to make 2 bootloader's sectors, MBR and GPT?

### DISCUSSION THREAD

**Discussion by Lyushen** [ventoy/Ventoy#739] [Q&A]
Like one hidden part with MBR and second one with GPT at the same time, or make like bootloader if booting from GPT not possible than boot from MBR... This idea of app is very cool, but would be perfect when you never need to rewrite your bootloader's sector

**Comment by chromer030** [comment]
MBR and GPT are partition tables, a disk can only have one partition table, partition table applies on whole disk and then partitions reside on it.
